### ISSUE TITLE
image has both width and fill properties on dev server

### ISSUE THREAD

**Issue by agoblet** [Niels-IO/next-image-export-optimizer#72]
I am upgrading to v13 of Next.js and v1 of this package. Building my static website works fine, but running the next dev server gives me errors when using `fill` images:

```
Error: Image with src "/_next/static/media/bol.4615d14b.png" has both "width" and "fill" properties. Only one should be used.
```

I am not specifying a width in the [image](https://github.com/agoblet/agoblet.github.io/blob/feature/upgrade-next-to-v13/pages/projects.tsx#L127):

```
<ExportedImage
  src={project.logoPath}
  alt={project.organization}
  useWebp
  fill
  style={{ objectFit: "scale-down", objectPosition: "right top" }}
/>
```

It only happens on the dev server, but it makes development much more cumbersome. It also happens to fill images in [other places](https://github.com/agoblet/agoblet.github.io/blob/feature/upgrade-next-to-v13/pages/index.tsx#L57). My other images, which don't use `fill`, do not cause any errors. I use the legacy component for fill images now as a workaround.

Any ideas on how this can be fixed?

**Comment by Niels-IO** [comment]
Hi @agoblet,

Thanks for letting me know. I think I know how to fix this. I will keep you posted!

**Comment by Niels-IO** [comment]
Hi @agoblet,

I fixed this now in v1.0.1. Please give it a try.

**Comment by agoblet** [comment]
works like a charm! thanks for the quick fix :)
